### DISCUSSION TITLE
Models Created from GGUF File Missing from api/models Endpoint

### DISCUSSION THREAD

**Discussion by chrisoutwright** [open-webui/open-webui#3730] [General]
### Issue Description
When creating a model from a GGUF file using the Ollama tool, the model appears in the `ollama list` command but is missing from the `api/models` endpoint after some time. This behavior persists despite changes to the system template.

### Steps to Reproduce

1. **Merge GGUF Files:**
   ```sh
   C:\Users\Chris>D:\llama-b3066-bin-win-avx512-x64\gguf-split --merge C:\Users\Chris\Downloads\Q4_K_M-00001-of-0002.gguf E:\openbiollm-llama-3_Q4_K_M.gguf
   ```
   **Output:**
   ```
   gguf_merge: C:\Users\Chris\Downloads\Q4_K_M-00001-of-00002.gguf -> E:\openbiollm-llama-3_Q4_K_M.gguf
   gguf_merge: reading metadata C:\Users\Chris\Downloads\Q4_K_M-00001-of-00002.gguf ...done
   gguf_merge: reading metadata C:\Users\Chris\Downloads\Q4_K_M-00002-of-00002.gguf ...done
   gguf_merge: writing tensors C:\Users\Chris\Downloads\Q4_K_M-00001-of-00002.gguf ...done
   gguf_merge: writing tensors C:\Users\Chris\Downloads\Q4_K_M-00002-of-00002.gguf ...done
   gguf_merge: E:\openbiollm-llama-3_Q4_K_M.gguf merged from 2 split with 723 tensors.
   ```

2. **Create Model in Ollama:**
   ```sh
   C:\Users\Chris>ollama create openbiollm-llama-3_Q4_K_M -f E:\openbiollm-llama-3_Q4_K_M.modelfile
   ```
   **Output:**
   ```
   transferring model data ⠋

   using existing layer sha256:8f3b57672da97273e31ea197ac93d6b44ec2c00af914c43c141f8a7571a3c844
   using existing layer sha256:2190828de961641d5a7b034d11c3e34f3a7e91e9ec195309770fb337231cc085
   using existing layer sha256:e9d486088426ca9362945844228c16a846e49f4d310c638fc79b60e780e46045
   using existing layer sha256:577073ffcc6ce95b9981eacc77d1039568639e5638e83044994560d9ef82ce1b
   using existing layer sha256:f2f08b422a621fe6eab6361c3cb1b66526f90b2db8ed4b54b8d5e06b8a5464f9
   writing manifest
   success
   ```

### Issue Details
- **Success:** The Ollama tool completes the model creation successfully.
- **Verification:** The `ollama list` command shows the new model.
- **Problem:** The new model is missing from the `api/models` endpoint after some hours, despite being listed initially.

### Original Model File:
```plaintext
FROM ./openbiollm-llama-3_Q4_K_M.gguf
TEMPLATE "{{ if .System }}system
{{ .System }}{{ end }}{{ if .Prompt }}user
{{ .Prompt }}{{ end }}assistant
{{ .Response }}"
PARAMETER stop 
PARAMETER stop 
PARAMETER stop 
PARAMETER num_keep 24
SYSTEM Format the reply in MarkDown format.
```

### Expected Behavior
The newly created model should be listed in both `ollama list` and `api/models` consistently, allowing it to be accessed via the API for applications like Open WebUI.

### Questions
1. **Model Name Capitalization:** Can the use of uppercase characters in the model name (e.g., `openbiollm-llama-3_Q4_K_M`) cause issues with the API listing?
2. **API Iterator:** What changes could cause the API iterator to skip over the newly created model?

### Environment
- **OS:** Windows 10
- **GPU:** Nvidia
- **CPU:** Intel
- **Ollama Version:** 0.1.48

## Environment

- **Open WebUI Version:** [e.g., 0.3.5]
- **Ollama (if applicable):** [e.g., 0.1.48]

- **Operating System:** [e.g., Windows 10, macOS Big Sur, Ubuntu 20.04]
- **Browser (if applicable):** [e.g., Chrome 100.0, Firefox 98.0]

## Reproduction Details

**Confirmation:**

- [x] I have read and followed all the instructions provided in the README.md.
- [ ] I am on the latest version of both Open WebUI and Ollama.
- [x] I have included the browser console logs.

## Logs and Screenshots

**Browser Console Logs:**
[Include relevant browser console logs, if applicable]

**Docker Container Logs:**
[Include relevant Docker container logs, if applicable]

**Screenshots (if applicable):**
[Attach any relevant screenshots to help illustrate the issue]

## Installation Method

Manual, Win 10

## Note

If the bug report is incomplete or does not follow the provided instructions, it may not be addressed. Please ensure that you have followed the steps outlined in the README.md and troubleshooting.md documents, and provide all necessary information for us to reproduce and address the issue. Thank you!

**Comment by WenjieDu** [comment]
I'm confronted with the same issue.

**Comment by qq527364584** [comment]
How to load custom models
